lib/polar_algs/polar_router.dart

```diff
@@ -0,0 +1,79 @@
+/// Polar Router
+/// Algorithm class that gets a route based on a polar plot
+import 'package:sail_routing_dart/polar_plotting/polar_plot.dart';
+import 'package:sail_routing_dart/shared/route_math.dart';
+import 'dart:math';
+import 'package:vector_math/vector_math.dart';
+const  minDiff = 10.0;
+class PolarRouter {
+  
+  PolarPlot _polarPlot;
+  
+  PolarRouter (PolarPlot plot) {
+    _polarPlot = plot;
+  }
+
+  /// Get route that has only been tested with wind at 180 degrees; 
+  /// should transform based on wind direction to handle other cases.
+  /// Arguments:
+  ///   (List<double>) start - Starting point in cartesian space
+  ///   (List<double>) end - End point where boat wants to go
+  ///   (double) windSpeed - Wind speed in knots
+  ///   (double) windDirection - Wind direction - Cardinal directions in degrees
+  /// Returns:
+  ///   (List<List<double>>) - List of points that specify where the boat should go   
+  List<List<double>> getTransRoute(
+    List<double> start,
+    List<double> end,
+    double windSpeed,
+    double windDirection
+  ) {
+    var route = List<List<double>>();
+    route.add(start);
+    // Get the route
```

Comment:
Should the rest of this just be put into a _getRoute() function then instead of leaving a comment?